x509-cert/src/crl.rs

```diff
@@ -75,7 +75,7 @@ pub struct RevokedCert<'a> {
 #[allow(missing_docs)]
 pub struct TbsCertList<'a> {
     pub version: Version,
-    pub signature: AlgorithmIdentifier<'a>,
+    pub signature: AlgorithmIdentifierRef<'a>,
```

Comment:
This should also allow `AlgorithmIdentifier<...>` (and in `Certificate`/`CertificateList`) too.
I can not come with the good enough argument at this moment.